web-test-runner.config.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env node */
+const { createSauceLabsLauncher } = require('@web/test-runner-saucelabs');
+
+const config = {
+  nodeResolve: true,
+  browserStartTimeout: 60000, // default 30000
```

Comment:
Consider using `120000` for iOS as that seems to be more reliable.

Reply:
Done.